apps/ewallet/lib/ewallet/release_tasks.ex

```diff
@@ -0,0 +1,88 @@
+defmodule EWallet.ReleaseTasks do
```

Comment:
Just making sure... this is part of this PR? Looks like it's a part of #312 or even its own PR?

Reply:
Nope, I forgot to delete it :\